### PR TITLE
Adding API Usage Examples Part 1

### PR DESCRIPTION
**Pull request by HaroldFS**
# This PR addresses the following:
- It is important for developers to have practical examples of how our API's can be used. 
- In the ReadMe file we should call out those APIs and point to them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,20 +55,58 @@ func fsIdentify(id: String, userInfo: [String: Any]) {
...
+    case viewCart
+    case addToCart
+    case checkout
+    case removeFromCart
+    case crash
+}
+```
+- Then we create a convenience method that takes a case and a dictionary to title the event and provide properties.
 ```
 func fsCreateEvent(event: Event, with dict: [String: Any]) {
```
Comment (sabrina-li) [inline README.md:71]: nit: fsCreateEvent maybe better named `fsSendEvent` or just `fsEvent`?

Comment (HaroldFS) [reply README.md:71]: Great idea! [CH Ticket](https://app.clubhouse.io/fullstory/story/121732/re-name-fscreateevent-to-fsevent) for later.

**Comment by van-fs** [comment]
I noticed a few typos in the readme.  I have a spell check editor extension for VSCode.  If you're using XCode and don't have a good markdown editor option, Typora is something I've used in the past and there's spell check built in.

**Comment by HaroldFS** [comment]
> I noticed a few typos in the readme. I have a spell check editor extension for VSCode. If you're using xcode and don't have a good markdown editor option, Typora is something I've used in the past and there's spell check built in.

OMG Typora Is amazing! And I used it for spell check and it also increased my productivity and how quickly I was able to complete this.

File: README.md [hunk 2]
```diff
@@ -36,6 +36,124 @@ The Shoppe is a super simple e-commerce application. Build and run the app on yo
...
+```swift
+fsLog(message: "User tried to checkout without confirmation.", level: .error)
+```
+
+Then the logs will show up the developer console:
+![LogExample](readmeImages/logExample.png "FullStory Logo")
+
+## Privacy
+
+For more info on how we inssure privacy is inforced on mobile, checkout this bionic blog on [Private by Default](https://bionic.fullstory.com/private-by-default-mobile-analytics/) by one of our Directors of Engineering at FullStory.
```
Comment (van-fs) [inline README.md]: inssure to ensure

File: README.md [hunk 3]
```diff
@@ -36,6 +36,124 @@ The Shoppe is a super simple e-commerce application. Build and run the app on yo
 - Fill out the form on the _Checkout_ view and click **Purchase**. The app persists locally all the information from this view. Please do **not** use your real information.
 - When clicking **Purchase** your information is validated and a Toast message will be shown. The purchase is successful if all fields passed the validate and your subtotal is greater than 0.
 
+## API Usage Examples
+
+`FullStoryManager.swift` (FSManager) is a file full of functions and enums that can make the FullStory API easier to apply to areas of your project. (Its an example of implementation for your app)
```
Comment (van-fs) [inline README.md]: Its to It's

File: README.md [hunk 4]
```diff
@@ -36,6 +36,124 @@ The Shoppe is a super simple e-commerce application. Build and run the app on yo
...
+```
+
+Then the logs will show up the developer console:
+![LogExample](readmeImages/logExample.png "FullStory Logo")
+
+## Privacy
+
+For more info on how we inssure privacy is inforced on mobile, checkout this bionic blog on [Private by Default](https://bionic.fullstory.com/private-by-default-mobile-analytics/) by one of our Directors of Engineering at FullStory.
+
+- We for the hardcoded values that represent the privacy settings we have an example of more enum utilization [here](https://github.com/fullstorydev/ios-shoppe-demo/blob/c534166901d71f0dace44f85aee053242dd25caf/swift/ios-shoppe-demo/ServiceManagers/FullStoryManager.swift#L32).
```
Comment (van-fs) [inline README.md]: "We for", sounds like something was left out.

**Comment by van-fs** [comment]
@HaroldFS Some higher level items we might want to consider:
- I feel like the description, design, and usage might be better done in the FullStoryManager.swift file as code comments.  The iOS Shoppe demo README may get verbose if we talk about implementation details like this.  You could alternatively comment in the file and then link to the file as a sample of how to wrap FullStory APIs to produce convenience methods.  Your code can offer a starting point.
- Having the comments in code supports another use case: you can simply send someone the file as an example of what to do or drop into a project.  This removes the need to have FullStoryManager AND the README to get the complete picture.
- If FullStoryManager provides sufficient convenience, should this be a separate package or part of FullStory core code?

**Comment by HaroldFS** [comment]
> @HaroldFS Some higher level items we might want to consider:
> 
> * I feel like the description, design, and usage might be better done in the FullStoryManager.swift file as code comments.  The iOS Shoppe demo README may get verbose if we talk about implementation details like this.  You could alternatively comment in the file and then link to the file as a sample of how to wrap FullStory APIs to produce convenience methods.  Your code can offer a starting point.
> * Having the comments in code supports another use case: you can simply send someone the file as an example of what to do or drop into a project.  This removes the need to have FullStoryManager AND the README to get the complete picture.
> * If FullStoryManager provides sufficient convenience, should this be a separate package or part of FullStory core code?

- Topic number one and two I think that makes a lot of sense. Maybe I'll move this documentation to in-line in that file.
- Topic three I think this is the eventual goal based on the things I have heard from Fred Potter when he got to review this FS manager implementation.

**Comment by HaroldFS** [comment]
I'm going to push this info back into the FS manager file.

**Pull request closed without merging**